### PR TITLE
fix(notification): support for longer messages

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: src/notification/__tests__/notification.scenario.js [hunk 1]
```diff
@@ -15,6 +15,10 @@ export const name = 'notification';
 export const component = () => (
   <React.Fragment>
     <Notification>Default info notification</Notification>
+    <Notification>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua
```
Comment (chasestarr) [inline src/notification/__tests__/notification.scenario.js]: does this handle the original request where there is a `<strong>` element in children?

Comment (gergelyke) [reply src/notification/__tests__/notification.scenario.js]: yeah, it does - do you want me to add it?

**Comment by jasonbarone** [comment]
LGTM 👍

File: src/notification/__tests__/notification.scenario.js (hunk 1 above)
Comment (chasestarr) [reply src/notification/__tests__/notification.scenario.js]: no big deal - just double-checking

**Pull request merged**